app/web_framework/cluster_master_application.py

```diff
@@ -19,6 +22,9 @@
 # pylint: disable=attribute-defined-outside-init
 #   Handler classes are not designed to have __init__ overridden.
 
+request_latency = Histogram('request_latency_seconds', 'Latency of HTTP requests in seconds')  # pylint: disable=no-value-for-parameter
```

Comment:
The `_MetricWrapper` decorator in the Prometheus client SDK seemed to be throwing off pylint.